definition/data/sheets/AuthorisationCaseEvent.json

```diff
@@ -26,6 +26,7 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "TestSupportUpdate", "UserRole": "caseworker-cmc", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "DirectionsQuestionnaireDeadline", "UserRole": "caseworker-cmc", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "SealedClaimUpload", "UserRole": "caseworker-cmc", "CRUD": "CR"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ReviewOrderUpload", "UserRole": "caseworker-cmc", "CRUD": "CR"},
```

Comment:
It should not have 'C' as this is just citizen action and caseworker should not be able to do this

Reply:
👍